src/I18n/I18n.php

```diff
@@ -29,6 +29,13 @@ class I18n
 {
 
     /**
+     * Default locale
+     *
+     * @var string
+     */
+    const DEFAULT_LOCALE = 'en_US';
```

Comment:
why make it a const, and not a public static variable

then I can change it without extending the class :)